Makefile

```diff
@@ -136,10 +136,14 @@ build-local-container: build ## Builds the provisioner container image using loc
 
 kind-load-bundles:
 	$(CONTAINER_RUNTIME) build $(TESTDATA_DIR)/bundles/plain-v0/valid -t testdata/bundles/plain-v0:valid
+	$(CONTAINER_RUNTIME) build $(TESTDATA_DIR)/bundles/plain-v0/dependent -t testdata/bundles/plain-v0:dependent
+	$(CONTAINER_RUNTIME) build $(TESTDATA_DIR)/bundles/plain-v0/provides -t testdata/bundles/plain-v0:provides
```

Comment:
Note: I could've used the valid testdata manifests and achieved the same behavior, but I was slightly concerned with breaking this test over time if we chose to refactor those manifests.

Reply:
Definitely - I can already see myself getting the itch to comb down the test data directory sizes at some point in the future.